src/example.ts

```diff
@@ -0,0 +1,29 @@
+import * as fileutils from './file-utils';
+
+class Example {
+    private readonly _folder: string;
```

Comment:
Even if the eslint rules don't prevent it, the NI style guide recommends to not use an underscore to prefix private class members. In this case, you can make them `public readonly` and remove the getter methods.